packages/components/docs/jest.md

```diff
@@ -98,8 +98,8 @@ has its own jest tests will currently need to mock the `LABKEY` object if that m
 
 ## Examples
 1. Testing of utility functions
-    1. [util/Date.spec.ts](../src/internal/util/Date.spec.ts)
-    1. [internal/app/utils.spec.ts](../src/internal/app/utils.spec.ts)
+    1. [util/Date.spec.ts](../src/internal/util/Date.test.ts)
```

Comment:
Here's an additional set of `spec.tsx` files that may be eligible to be converted to `test.ts` (dropping the "x" in .tsx):

```sh
# find . -name "*.spec.tsx" | xargs grep  -L -e  "mount\|shallow\|renderer" *
./internal/app/utils.spec.tsx
./internal/util/similaritySortFactory.spec.tsx
./internal/util/messaging.spec.tsx
./internal/components/forms/utils.spec.tsx
./internal/components/forms/input/InputRenderFactory.spec.tsx
./internal/components/productnavigation/utils.spec.tsx
./internal/components/samples/utils.spec.tsx
./internal/components/base/models/User.spec.tsx
./internal/components/base/SVGIcon.spec.tsx
./public/QueryModel/grid/actions/Filter.spec.tsx
./public/QueryModel/grid/actions/Sort.spec.tsx
```

If converted then I'd recommend perusing the imports in each to make sure they are up to date.